File: TextIndexApi/src/main/java/com/hideakin/textsearch/index/model/UpdatePreferenceRequest.java
package com.hideakin.textsearch.index.model;

public class UpdatePreferenceRequest {

	private String name;
	
	private String value;
	
	public UpdatePreferenceRequest() {
	}

	public UpdatePreferenceRequest(String name, String value) {
		this.name = name;
		this.value = value;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}

}

File: TextSearchNet/model/UpdatePreferenceRequest.cs
﻿using Newtonsoft.Json;

namespace com.hideakin.textsearch.model
{
    internal class UpdatePreferenceRequest
    {
        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("value")]
        public string Value { get; set; }

        public UpdatePreferenceRequest()
        {
        }

        public UpdatePreferenceRequest(string name, string value)
        {
            Name = name;
            Value = value;
        }
    }
}
